js/views/chat/Conversation.js

```diff
@@ -425,7 +425,6 @@ export default class extends baseVw {
     if (this._isOpen) return;
     this._isOpen = true;
     getBody().addClass('chatConvoOpen');
-    this.$messageInput.focus();
```

Comment:
Why was this removed? Don't we want to focus the message input field when you open the convo window?